docs/docs-ref-autogen/office/office.settings.yml

```
@@ -43,8 +43,8 @@ items:
       Adds an event handler for the settingsChanged event.
 
 
-      Important: Your add-in's code can register a handler for the settingsChanged event when the add-in is running with
-      any Excel client, but the event will fire only when the add-in is loaded with a spreadsheet that is opened in
+      **Important**: Your add-in's code can register a handler for the settingsChanged event when the add-in is running
+      with any Excel client, but the event will fire only when the add-in is loaded with a spreadsheet that is opened in
       Excel Online, and more than one user is editing the spreadsheet (co-authoring). Therefore, effectively the

```

```suggestion
      Excel Online and more than one user is editing the spreadsheet (co-authoring). Therefore, effectively the
```
I don't think this comma is needed?